<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Umut Fitness</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Umut Fitness'a Hoşgeldiniz Lütfen Giriş Yapın</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kullaniciadi">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Kullanıcı Adı : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kullaniciadi">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_parola">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Parola : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_parola">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>

   <widget class="QPushButton" name="btn_giris">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
       <property name="styleSheet">
        <string notr="true">background-color: green; color: white;</string>
    </property>
   </widget>
      <widget class="QPushButton" name="btn_kayitol">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue; color: black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
